Assets/Plugins/Source/Core/Utility/JsonUtility.cs

```diff
@@ -18,13 +18,11 @@
  * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
- */
+*/
 
 namespace PlayEveryWare.EpicOnlineServices.Utility
 {
-    using Newtonsoft.Json;
-    using Newtonsoft.Json.Linq;
-    using System;
+    using Codice.Client.ChangeTrackerService;
```

Comment:
I wonder what is this for. Maybe another resharper thing

Reply:
I believe this needs to be removed.